utils/graph.h

```diff
@@ -9,12 +9,20 @@
 #include "utils/list.h"
 #include "utils/rbtree.h"
 
+struct graph_time_stat {
+	uint64_t sum;
+	uint64_t avg;
+	uint64_t min;
+	uint64_t max;
+};
+
 struct uftrace_graph_node {
 	uint64_t addr;
 	char *name;
 	int nr_edges;
 	int nr_calls;
-	uint64_t time;
+	struct graph_time_stat total_time;
+	struct graph_time_stat self_time;
```

Comment:
I think you can add self_time in the next commit.

Reply:
@namhyung 
Do you mean I should split into each commit total_time and self_time?